Assets/LeapMotionModules/Interaction/Examples/Basic Example/SimplePrimaryHoverGlow.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Leap.Unity;
+using Leap.Unity.RuntimeGizmos;
+using Leap.Unity.UI.Interaction;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(InteractionBehaviour))]
+public class SimplePrimaryHoverGlow : MonoBehaviour, IRuntimeGizmoComponent {
+
+  private InteractionBehaviour _intObj;
+
+  private BoxCollider _collider;
+
+  void Start() {
+    _intObj = GetComponent<InteractionBehaviour>();
+
+    _collider = GetComponent<BoxCollider>();
+  }
+
+  void Update() {
+    if (_intObj.isGrasped) {
+      Turn(Color.red);
+    }
+    else {
+      if (_intObj.isPrimaryHovered) {
+        // var hand = _intObj.primaryHoveringHand;
+        TurnWhite();
+      }
+      else {
+        TurnBlack();
+      }
+    }
+  }
+
+  private Material _matInstance;
+
+  private void TurnWhite() {
```

Comment:
I think having dedicated methods for turning black and white is a little silly.  Even more so that they are not just wrappers around Turn(Color.black) ect...